md/performance-tuning.md

```diff
@@ -31,22 +31,38 @@ Two key definitions:
 
 Performance tuning checklist of best practises:
 
-* [Access Engine APIs](#engine_access) in Local mode whenever possible.
-* Make sure that the [network](#hardware) between your client and the Bonita BPM Engine server is fast if you [access the APIs remotely](#remote).
-* Install your [Database](#db) on a powerful [machine](#hardware) (hardware that meets to your DB vendor requirements: sufficient memory, powerful CPU, and fast I/O).
-* Make sure that the [network](#hardware) between your Bonita BPM Engine server and your database server is very fast.
-* Set the [Work service](#work_service) threadpool and [Connector service](#connector_service) threadpool to the same size.
-* Set the maximum [DB connections](#db_connections) for the bonitaDS [datasource](#datasource_settings) to the desired processing capacity number of parallel threads.
-* Configure a suitable [sequence manager](#seq_mgr) range size for your typical process designs and expected volume. 
-* Make sure that the maximum [DB connections](#db_connections) for sequenceManagerDS [datasource](#datasource_settings) is more than 1 and is appropriate for the SequenceManager range size configuration.
-* Tune the [application cache](#app_cache) to your hardware capabilities (if you have more memory available, increase the cache size).
-* Set the basic [JVM options](#jvm) to make it fast and well-sized.
-* Optimize your [database configuration](#db) for your most frequent usage (read or write) and the level of robustness that you want.
-* Configure your [transaction manager](#tm) for the level of robustness that you want.
-* Tune the [log levels](#logs) of Bonita BPM Engine and all its dependencies.
-* Design your [processes](#process_design) following the best practises.
-* Add a reasonable number of well-developed [event handlers](#event_handlers).
-* Tuned the Bonita BPM Engine [cron jobs](#cron) for your needs.
+- [Performance tuning](#performance-tuning)
```

Comment:
I don't understand why you changed this entire section ...

Reply:
I have big fingers!